.github/workflows/admob.yml

```diff
@@ -0,0 +1,39 @@
+name: AdMob
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    name: build
+    runs-on: macOS-latest
+    strategy:
+        matrix:
+          swift-suffix: ['', 'Swift']
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Build
+        run: |
+          cd admob
```

Comment:
Issues are generally easier to diagnose for with separate named steps for the commands.  Especially to distinguish between setup and running.

Reply:
Done.